pkgs/development/tools/misc/babeltrace/common.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchzip
+, autoreconfHook
+, pkg-config
+, glib
+, elfutils
+
```

Comment:
I'd not separate the `pname` and `version` from the other arguments...

Reply:
Yes, this simplify things.